pschlatt/Tripedia
# Directions matrix

#### What's this PR do?

Reformats trip creation to return all possible categories on route that user has created with origin and destination so FE can filter through options selected and return pins as requested.

#### Where should the reviewer start?

Oh boy, likely with the spec and trace through the controller to yelp facade to check out formatting.

#### How should this be manually tested?

You can run rspec to see how response will be returned, *but* I likely will create a fixture to run against so yelp api is not pinged a million times.

#### Any background context you want to provide?

Nope, you were there for the entire struggle, I believe we all have a solid understanding of the trip creation route now!

#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Questions:
